apps/web/src/composables/users.ts

```diff
@@ -6,19 +6,49 @@ import { ProviderString } from '@/types/ProviderString'
 import { User } from '@/interfaces/User'
 import { onMounted, ref } from 'vue'
 
+const { _getAuthBaseUrl } = useEnvironment()
+
 const user = ref<User>({
     id: '0xd557a5745d4560B24D36A68b52351ffF9c86A212',
     accounts: {
         MetaMask: ['0xd557a5745d4560B24D36A68b52351ffF9c86A212']
     } as Record<ProviderString, string[]>,
+    primaryAccount: '0xd557a5745d4560B24D36A68b52351ffF9c86A212',
     pools: []
 })
 
 export default function useUsers () {
 
-    const { ethereumURL } = useEnvironment()
-    const { getUserBalance, getUserPools } = useWallet()
-    const { ssv } = useSSV()
+    // Commenting out for now to avoid errors
```

Comment:
What errors?

Reply:
Mentioned the errors in Slack. Did you run it and try commenting out?